app/lib/bk/compat/parsers/bitrise/github_release.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'uri'
+
+module BK
+  module Compat
+    module BitriseSteps
+      # Handles the translation of github-release
+      class Translator
+        def translate_github_release(inputs)
+          raise 'Invalid github-release step configuration!' unless validate_required_params(inputs)
+
+          release_config = extract_release_config(inputs)
+          repo_path = parse_repo(release_config['repository_url'])
+
+          commands = []
+          add_upload_commands(commands, release_config, repo_path) if release_config['files_to_upload']
+
+          BK::Compat::CommandStep.new(commands: commands)
+        end
+
+        private
+
+        def validate_required_params(inputs)
+          %w[api_token username repository_url tag commit name].all? { |key| inputs.include?(key) }
+        end
+
+        def extract_release_config(inputs)
+          defaults = {
+            'body' => '',
+            'draft' => 'yes',
+            'pre_release' => 'no',
+            'api_base_url' => 'https://api.github.com',
+            'upload_base_url' => 'https://uploads.github.com',
+            'generate_release_notes' => 'no'
+          }
+
+          defaults.merge(inputs)
+        end
+
+        def parse_repo(url)
+          url = url.chomp('.git')
+
+          if url.start_with?('git@')
+            url = url.sub(':', '/')
+            url = "ssh://#{url}"
+          end
+
+          uri = URI.parse(url)
+          path_parts = uri.path.split('/').reject(&:empty?)
+
+          raise 'Invalid repository URL!' if path_parts.size < 2
+
+          owner, repo_name = path_parts[-2..]
+          [owner, repo_name]
+        end
```

Comment:
you are returning an array here and not a string which means the result looks something like:

```
    https://uploads.github.com/repos/["example", "example-repository"]/releases/assets?name=${APP_NAME}-debug-androidTest.apk
```

You should use the fact that the URI module already parsed it for you:

```suggestion
          uri = URI.parse(url)
          uri.path
        end
```